test/app.js

```diff
@@ -213,65 +217,28 @@ router.get('/:key', (req, res) => {
     } finally {
 
     }
-    signale.info(`Requested http://localhost:3030/${key}`);
-    res.render(`${key}/index`, Object.assign(GLOBALS, { id: key, component: getStarterData(), data, libs: getLibs(req.query.lib) }));
```

Comment:
Is the `libs` no longer needed?

Reply:
It was never being used - it should have been living on a branch as it wasn't a fully developed feature.